src/Queue/Failed/MongoFailedJobProvider.php

```diff
@@ -75,12 +76,43 @@ public function find($id)
     /**
      * Delete a single failed job from storage.
      *
-     * @param mixed $id
+     * @param string $id
      *
      * @return bool
      */
     public function forget($id)
     {
         return $this->getTable()->where('_id', $id)->delete() > 0;
     }
+
+    /**
+     * Get the IDs of all of the failed jobs.
+     *
+     * @param  string|null $queue
+     *
+     * @return array
```

Comment:
Since there are no type requirements on the `_id` field, this is essentially a list of mixed values, correct? Any reason to use not to use the [list type](https://phpstan.org/writing-php-code/phpdoc-types#lists)?